e2e/testenv/infra/cmd/setup.go

```diff
@@ -70,9 +70,13 @@ var setupCmd = &cobra.Command{
 			"rev-parse",
 			"--show-toplevel",
 		)
-		repoRoot, err := repoRootCmd.Output()
+		cwd, err := os.Getwd()
 		if err != nil {
-			fail(fmt.Errorf("get repo root: %v", err))
+			fail(err)
+		}
+		repoRoot, err := repoRootCmd.CombinedOutput()
+		if err != nil {
+			fail(fmt.Errorf("get repo root in %s: %v: %s", cwd, err, repoRoot))
```

Comment:
nit: I think this error message could be more readable with the error placed last, as the error may be longer than other fields. Eg. something like:
```suggestion
			fail(fmt.Errorf("get repo root in %s: result %q, error %v", cwd, err, repoRoot))
```